demux/utils/novaseq_samplesheet.py

```diff
@@ -48,10 +59,10 @@ def header(self) -> list:
     @staticmethod
     def get_dummy_samplesheet_sample(
         flowcell: str, dummy_index: str, lane: int, name: str
-    ) -> str:
+    ) -> Dict[Union[str, Any], Union[Union[str, int], Any]]:
```

Comment:
The name of this function is a bit misleading, looks like a dictionary with sample information is returned and not just a name

Reply:
improved name suggestion in https://github.com/Clinical-Genomics/demultiplexing/pull/146/commits/aba742c51a4ac99eb9d4f81cf0cc141c27d6eaa0